components/connector-service/internal/monitoring/collector/summarycollector_test.go

```diff
@@ -0,0 +1,54 @@
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestNewSummaryCollector(t *testing.T) {
+
+	t.Run("should create and register summary collector", func(t *testing.T) {
```

Comment:
Consider renaming to "should create summary collector"

Reply:
Done